File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/service/creation/entity/PostmanService.java
package org.javamaster.invocationlab.admin.service.creation.entity;

import org.javamaster.invocationlab.admin.service.GAV;

import java.util.List;

/**
 * @author yudong
 */
public interface PostmanService {

    String getCluster();

    String getServiceName();

    GAV getGav();

    List<InterfaceEntity> getInterfaceModels();

    boolean getLoadedToClassLoader();

    void setLoadedToClassLoader(boolean load);
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/service/creation/PostmanService.kt
package org.javamaster.invocationlab.admin.service.creation

import org.javamaster.invocationlab.admin.service.GAV
import org.javamaster.invocationlab.admin.service.creation.entity.InterfaceEntity

/**
 * @author yudong
 */
interface PostmanService {
    fun getCluster(): String

    fun getServiceName(): String

    fun getGav(): GAV

    fun getInterfaceModels(): List<InterfaceEntity>

    fun getLoadedToClassLoader(): Boolean

    fun setLoadedToClassLoader(b: Boolean)
}
